app/templates/components/twilio-make-call.hbs

```diff
@@ -0,0 +1,11 @@
+<div id="twilio_token" data-token=""></div>
```

Comment:
@swatijadhav mentioned this in another comment - do we need this token inside the dom? Can we just store it as a variable in the component and use it to initialize the Twilio.Device directly?